bgpd/bgp_network.c

```diff
@@ -371,7 +387,7 @@ int bgp_tcp_mss_set(struct peer *peer)
 			 * one peer that is in passive mode. Otherwise, TCP MSS
 			 * is set per socket via bgp_connect().
 			 */
-			if (CHECK_FLAG(peer->flags, PEER_FLAG_PASSIVE))
+			if (min_mss != 0)
```

Comment:
Default and non-passive neighbors have by default 0 (= 1500, as the kernel has). What's the real issue here?

Reply:
When the bgp neighbor in passive mode, the tcp socket of neighbor always created by accept, but when the neighbor not in passive mode, the tcp socket of neighbr may also created by accept.
So if the neighbor config tcp-mss as 1000 but not config passive mode, when the neighbor establish by bgp_accept, the tcp-mss also used the default (1500 in linux) tcp-mss.

Reply:
Then please fix the comment for this check also.